internal/plugins/plugdeps/plugdeps_test.go

```diff
@@ -52,7 +52,7 @@ func Test_List_On(t *testing.T) {
 	app := meta.New(os.TempDir())
 
 	p := ConfigPath(app)
-	r.NoError(os.MkdirAll(filepath.Dir(p), 0755))
+	r.NoError(os.MkdirAll(filepath.Dir(p), 0o755))
```

Comment:
Found that 0o means octal. For some reason I've always been using 0XXX and thought it would take it as octal without specifying the `o`